### PR TITLE
Replaced ECG "Forbidden functions" file_exists/file_get_contents

### PR DESCRIPTION
**Pull request by sreichel**

### CODE REVIEW

**Comment by sprankhub** [comment]
Thanks for your contribution! The changes look good, although it is a bit hard to follow due to your code style changes. I did not test the changes.

I think this should be merged after it has been tested by someone and #287 has been handled.

**Comment by riconeitzel** [comment]
reviewed and approved

**Comment by riconeitzel** [comment]
@sreichel feedback: you forgot to include the filename after the check in 3 cases which resulted in errors not getting the file content loaded. Fixed it, but wanted to let you know #improvingAndStuff 😃

**Comment by sreichel** [comment]
@riconeitzel thx for fixing it!
